testresults/summary.md

```diff
@@ -0,0 +1,60 @@
+# Ballerina Performance Test Results
+
+During each release, we execute various automated performance test scenarios and publish the results.
+
+| Test Scenarios | Description |
+| --- | --- |
+| Passthrough HTTP service (h1c -> h1c) | An HTTP Service, which forwards all requests to an HTTP back-end service. |
+
+Our test client is [Apache JMeter](https://jmeter.apache.org/index.html). We test each scenario for a fixed duration of
+time. We split the test results into warmup and measurement parts and use the measurement part to compute the
+performance metrics.
+
+A majority of test scenarios use a [Netty](https://netty.io/) based back-end service which echoes back any request
+posted to it after a specified period of time.
+
+We run the performance tests under different numbers of concurrent users, message sizes (payloads) and back-end service
+delays. And also we can change the infrastructure of the ballerina server.
+
+The main performance metrics:
+
+1. **Throughput**: The number of requests that the Ballerina service processes during a specific time interval (e.g. per second).
+2. **Response Time**: The end-to-end latency for an operation of invoking a Ballerina service. The complete distribution of response times was recorded.
+
+In addition to the above metrics, we measure the load average and several memory-related metrics.
+
+The following are the test parameters.
+
+| Test Parameter | Description | Values |
+| --- | --- | --- |
+| Scenario Name | The name of the test scenario. | Refer to the above table. |
+| Heap Size | The amount of memory allocated to the application | 2G |
+| Concurrent Users | The number of users accessing the application at the same time. | 50, 100 |
+| Message Size (Bytes) | The request payload size in Bytes. |  |
+| Back-end Delay (ms) | The delay added by the back-end service. |  |
+
+The duration of each test is **900 seconds**. The warm-up period is **300 seconds**.
+The measurement results are collected after the warm-up period.
+
+The specific memory and cpu values(https://docs.amazonaws.cn/en_us/AmazonECS/latest/APIReference/API_TaskDefinition.html) were used to install Ballerina.
```

Comment:
Let's remove this and add the memory and CPU used for Ballerina docker. Something like below.
```suggestion
Ballerina Test CPU: **4**
Ballerina Test Memory: **8GB**
```